internal/repository/hook.go

```diff
@@ -42,12 +42,12 @@ func (r *Repository) UpdateHook(hookType HookType, content []byte, force bool) e
 	}
 
 	repoRoot := tree.Filesystem.Root()
-	hookFolder := path.Join(repoRoot, ".git", "hooks")
+	hookFolder := filepath.Join(repoRoot, ".git", "hooks")
```

Comment:
:+1:

Reply:
Side note: once the gitinterface PRs land, I'll update this to use the `repo` context to derive git_dir correctly.